src/cloudsc2_nl_loki/dwarf_cloudsc.F90

```diff
@@ -101,6 +102,7 @@ PROGRAM DWARF_CLOUDSC
 ! Instead we do this similar computation involving values already available
 DO JK=1,GLOBAL_STATE%KLEV
   YRECLD%CETA(JK)= GLOBAL_STATE%PAP(1,JK,1)/GLOBAL_STATE%PAPH(1,GLOBAL_STATE%KLEV+1,1)
+  LCETA(JK)=YRECLD%CETA(JK)
```

Comment:
I think this is not needed. We've encountered this in other contexts, where the problem is that ALLOCATABLE components of a derived type object are not copied to device with the rest of the components unless explicitly forced via `!$acc create copyin` or `!$acc data` regions. Since Loki's insertion of the OpenACC offload regions accepts hand-rolled pragmas in addition to the auto-generated ones, we can probably add the missing pragma in the driver directly, so we don;t have to do all this back-bending here.

FWIW, this will hopefully fixed in future Loki-SCC versions, where Loki will add the missing copies explicitly, but that is future work for now.

Reply:
I can have a quick go at this and see how to best handle this. I'll ping once I can say more...